nixos/modules/services/desktops/gnome3/gnome-settings-daemon.nix

```diff
@@ -0,0 +1,45 @@
+# GNOME Settings Daemon
+
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.gnome3.gnome-settings-daemon;
+
+in
+
+{
+
+  ###### interface
+
+  options = {
+
+    services.gnome3.gnome-settings-daemon = {
+
+      enable = mkEnableOption "GNOME Settings Daemon.";
+
+      # There are many forks of gnome-settings-daemon
+      package = mkOption {
+        type = types.package;
+        default = pkgs.gnome3.gnome-settings-daemon;
+        description = "Which gnome-settings-daemon package to use.";
+      };
+
+    };
+
+  };
+
+
+  ###### implementation
+
+  config = mkIf cfg.enable {
+
+    environment.systemPackages = [ cfg.package ];
```

Comment:
Do we need this? Neither Elementary or GNOME did this.

Reply:
We did, it was just hidden:
https://github.com/NixOS/nixpkgs/blob/9dd81f9389c886226796613794776b712c1255ad/pkgs/desktops/pantheon/default.nix#L30
https://github.com/NixOS/nixpkgs/blob/2fcb11a2442b27834db87b2f62511037c31b5e36/pkgs/desktops/gnome-3/default.nix#L30

Reply:
Why did we not remove it from there?

Reply:
I think we only need `share/polkit-1/actions` linked. We can use `pathsToLink` for that right?

Reply:
That would already be handled by the polkit module.

Reply:
Though I think maybe it still should be in `gnome3.corePackages` because people may depend on it being there.

Reply:
How do you mean?

Reply:
using
```nix
environment.systemPackages = [ ... ] ++ pkgs.gnome3.corePackages;
```
and not using our gnome3 module.

But that really doesn't matter, I'll remove it from those lists.

Reply:
Hmm, I do not consider that part of public interface but Nix does not really help us here. We ought to create a minimal GNOME profile. Something like https://gitlab.com/yegortimoshenko/dotfiles/blob/master/modules/services/gnome3.nix